eclipse-scout-core/src/tabbox/SimpleTabAreaLayout.ts

```diff
@@ -133,16 +198,14 @@ export class SimpleTabAreaLayout extends AbstractLayout {
     $tab = $tab.clone().addClass('selected'); // Non selected items have a margin, selected ones don't -> we need to get the width incl. margin
     let tabAreaClasses = this.tabArea.$container.attr('class');
     let tabItemClasses = $tab.attr('class');
-    if (this.tabWidth === null) {
-      this.tabWidth = styles.getSize([tabAreaClasses, tabItemClasses], 'width', 'width', 0);
+    if (this.tabSize === null) {
```

Comment:
What do you think about a pattern like this:

```suggestion
    this._initTabSize([tabAreaClasses, tabItemClasses], this._horizontalTabs);
```

Including the function `_initTabSize`

```
  protected _initTabSize(cssClasses: string[], horizontal: boolean) {
    if (this.tabSize !== null) {
      return;
    }

    if (horizontal) {
      this.tabSize = styles.getSize(cssClasses, 'width', 'width', 0);
      return;
    }

    this.tabSize = styles.getSize(cssClasses, 'height', 'height', 0);
  }
```

It is a little more code, but it eliminates the necessity to pass all these callbacks. So I think its easier to read, debug and maintain.